### DISCUSSION TITLE
awesome-ribbit-network

### DISCUSSION THREAD

**Discussion by josuah** [Ribbit-Network/ribbit-network-frog-hardware#231] [General]
Here is a thread to collect useful links that might interest people from inside and outside the project. I will do my best to collect the links submitted down there into categories there:

## Application notes

https://edgecollective.io/img/co2/CD_AN_SCD30_Field_Calibration_D2.pdf
https://www.sensirion.com/media/documents/33C09C07/620638B8/Sensirion_SCD30_Field_Calibration.pdf
https://learn.adafruit.com/adafruit-scd30/field-calibration

## Data visualizers

https://ourworldindata.org/co2-emissions-metrics

## Sensor research and scientific papers

https://canair.io/docs/co2_comparative.html
https://www.airgradient.com/blog/co2-sensors-photo-acoustic-vs-ndir/
https://www.airgradient.com/blog/performance-of-low-cost-co2-sensors-outdoors/
https://amt.copernicus.org/articles/13/3815/2020/
https://www.science.org/doi/10.1126/sciadv.adh2391

## Products

https://www.infineon.com/cms/en/product/sensor/co2-sensors/
https://www.adafruit.com/product/4415 - Adafruit Mini GPS PA1010D - UART and I2C - STEMMA QT
https://www.adafruit.com/product/4494 - Adafruit DPS310 Precision Barometric Pressure / Altitude Sensor - STEMMA QT / Qwiic
https://www.adafruit.com/product/4867 - Adafruit SCD-30 - NDIR CO2 Temperature and Humidity Sensor - STEMMA QT / Qwiic
https://www.adafruit.com/product/5187 - Adafruit SCD-40 - True CO2, Temperature and Humidity Sensor - STEMMA QT / Qwiic
https://www.youtube.com/watch?v=ioWGK-l3uOw - Adafruit EYE on NPI: Amphenol T6793 CO2 Sensor Module
https://www.co2meter.com/collections/1-percent-co2-sensor
https://www.co2meter.com/products/k-30-co2-sensor-module
https://www.vaisala.com/en/products/instruments-sensors-and-other-measurement-devices/instruments-industrial-measurements/gmp252
https://www.elichens.com/elos-outdoor-air-quality-station
https://openflexure.org/projects/microscope/

## Events and publications about Ribbit

https://gml.noaa.gov/annualconference/abs.php?refnum=11-230417-A
https://www.linkedin.com/posts/seeedstudio_makers4sdgs-beaglebone-co2-activity-6936980642723151872-KZcj/
https://twitter.com/seeedstudio/status/1531815710212599808?t=sxUcB_Ej_T_1fj1IgDFEdg
https://www.hackster.io/stevenpest/seattleribbitnetwork-e550e2
https://www.youtube.com/watch?v=3GqF9lK8hqI - Keenan Johnson, Ribbit Network // Hackster Café

## Event and publications by Ribbit

https://ribbitnetwork.notion.site/FAQ-Frog-sensor-edf42fd302a34430abedff4e1df3da45
https://www.linkedin.com/posts/activity-7066461551670542336-KL1m/
https://ribbit.hudsoncreativegroup.com/

## News articles

https://www.newscientist.com/article/2317175-we-can-now-tell-how-much-co2-in-the-air-is-due-to-fossil-fuel-burning/
https://www.nytimes.com/interactive/2022/12/02/climate/mauna-loa-eruption-keeling-curve.html
https://arc-anglerfish-washpost-prod-washpost.s3.amazonaws.com/public/XQSGP4YPJFFTJI6NX24OVYZFKI.jpg

## Initiatives, organizations, solution providers

https://www.noaa.gov/weather
https://www.icos-cp.eu/projects/icos-cities
https://www.icos-cp.eu/
https://www.decentlab.com/
https://atmosphere.ei.tum.de/
https://www.neonscience.org/resources/research-support/field-site-coordination
https://cambridgemosscow.org/
https://www.airgradient.com/
https://en.origins.earth/
https://www.cloudcarbonfootprint.org/

**Comment by josuah** [comment]
Unsorted links from the chat

**Reply by josuah** [reply]
https://edgecollective.io/img/co2/CD_AN_SCD30_Field_Calibration_D2.pdf
https://ourworldindata.org/co2-emissions-metrics
https://www.science.org/doi/10.1126/sciadv.adh2391
https://www.infineon.com/cms/en/product/sensor/co2-sensors/
https://edgecollective.io/img/co2/CD_AN_SCD30_Field_Calibration_D2.pdf

**Reply by josuah** [reply]
https://gml.noaa.gov/annualconference/abs.php?refnum=11-230417-A
https://www.adafruit.com/product/4415
https://www.adafruit.com/product/4494
https://www.adafruit.com/product/4867
https://www.newscientist.com/article/2317175-we-can-now-tell-how-much-co2-in-the-air-is-due-to-fossil-fuel-burning/
https://canair.io/docs/co2_comparative.html
https://www.youtube.com/watch?v=ioWGK-l3uOw
https://www.nytimes.com/interactive/2022/12/02/climate/mauna-loa-eruption-keeling-curve.html
https://www.sensirion.com/media/documents/33C09C07/620638B8/Sensirion_SCD30_Field_Calibration.pdf
https://learn.adafruit.com/adafruit-scd30/field-calibration
https://ribbitnetwork.notion.site/FAQ-Frog-sensor-edf42fd302a34430abedff4e1df3da45
https://www.airgradient.com/blog/co2-sensors-photo-acoustic-vs-ndir/
https://www.co2meter.com/products/k-30-co2-sensor-module
https://www.co2meter.com/collections/1-percent-co2-sensor
https://www.linkedin.com/posts/activity-7066461551670542336-KL1m
https://www.decentlab.com/
https://www.vaisala.com/en/products/instruments-sensors-and-other-measurement-devices/instruments-industrial-measurements/gmp252
https://www.icos-cp.eu/projects/icos-cities
https://atmosphere.ei.tum.de/
https://www.icos-cp.eu/
https://arc-anglerfish-washpost-prod-washpost.s3.amazonaws.com/public/XQSGP4YPJFFTJI6NX24OVYZFKI.jpg
https://www.airgradient.com/blog/performance-of-low-cost-co2-sensors-outdoors/
https://ribbit.hudsoncreativegroup.com/

**Reply by josuah** [reply]
https://amt.copernicus.org/articles/13/3815/2020/
https://www.hackster.io/stevenpest/seattleribbitnetwork-e550e2
https://www.neonscience.org/resources/research-support/field-site-coordination
https://cambridgemosscow.org/
https://www.airgradient.com/open-airgradient/instructions/
https://www.adafruit.com/product/5187#technical-details
https://www.youtube.com/watch?v=3GqF9lK8hqI
https://en.origins.earth/
https://www.elichens.com/elos-outdoor-air-quality-station
https://www.cloudcarbonfootprint.org/
https://twitter.com/seeedstudio/status/1531815710212599808?t=sxUcB_Ej_T_1fj1IgDFEdg
https://www.linkedin.com/posts/seeedstudio_makers4sdgs-beaglebone-co2-activity-6936980642723151872-KZcj/?utm_source=linkedin_share&utm_medium=member_desktop_web

**Comment by josuah** [comment]
Spontaneously contributed, personal readings

**Reply by josuah** [reply]
https://openflexure.org/projects/microscope/

**Comment by keenanjohnson** [comment]
Thanks @josuah
